R/fullREMIND.R

```diff
@@ -15,7 +13,7 @@
 #' fullREMIND()
 #' }
 #'
-fullREMIND <- function(rev = 0) {
```

Comment:
did this rev argument has any functionality in retrieveData, which we are only not using?